highfidelity/hifi
# Ensure monitor never shows frozen SpectatorCamera preview

This PR ensures that, no matter what combination of Spectator Camera settings you have, you will never see a frozen, single-frame preview of the Spectator Camera.

**Test Plan:**
1. Go into HMD mode. Ensure that the "Disable Preview" setting in the Display menu is *checked*.
2. Open the Spectator Camera tablet app. If it isn't already, set the "MONITOR SHOWS" switch to "HMD Preview".
3. Enable the Spectator Camera
4. Peek behind your headset and verify that the monitor displays the "READY" image (and not a preview of what you're looking at in your HMD)
5. Move the "MONITOR SHOWS" switch to "Camera View"
6. Peek behind your headset and verify that the monitor displays what the Spectator Camera is capturing
7. Move the "MONITOR SHOWS" switch to "HMD Preview"
8. Peek behind your headset and verify that the monitor displays the "READY" image
9. In Interface's menu, *uncheck* the "Disable Preview" setting in the Display menu
10. Peek behind your headset and verify that the monitor is displaying a preview of what you're looking at in your HMD
11. Move the "MONITOR SHOWS" switch to "Camera View"
12. Peek behind your headset and verify that the monitor displays what the Spectator Camera is capturing
13. Uncheck the "Camera On" checkbox in the Spectator Camera tablet app
14. Peek behind your headset and verify that the monitor is displaying a preview of what you're looking at in your HMD
15. Check the "Camera On" checkbox
16. Peek behind your headset and verify that the monitor displays what the Spectator Camera is capturing
17. In Interface's menu, *check* the "Disable Preview" setting in the Display menu
18. Peek behind your headset and verify that the monitor displays what the Spectator Camera is capturing
19. Move the "MONITOR SHOWS" switch to "HMD Preview"
20. Switch into Desktop mode and verify that your monitor displays a first person view from your avatar (not from the Spectator Camera)
